src/components/Modals/ENSMapAddressModal/ENSMapAddressModal.tsx

```diff
@@ -0,0 +1,61 @@
+import { useCallback, useEffect, useState } from 'react'
+import { Button, Close, Field, Message } from 'decentraland-ui'
+import Modal from 'decentraland-dapps/dist/containers/Modal'
+import ethereumImg from '../../../icons/ethereum.svg'
+import { Props } from './ENSMapAddressModal.types'
+import styles from './ENSMapAddressModal.module.css'
+import { t } from 'decentraland-dapps/dist/modules/translation/utils'
+
+export default function EnsMapAddressModal(props: Props) {
+  const { isLoading, error, onClose, onSave } = props
+  const [address, setAddress] = useState<string>('')
+
+  const handleSave = useCallback(() => {
+    onSave(address)
+  }, [address, onSave])
+
+  const handleAddressChange = useCallback(
+    (_evt, { value }) => {
+      setAddress(value)
+    },
+    [setAddress]
+  )
+
+  useEffect(() => {
+    document.getElementById('address-input')?.focus()
+  }, [])
+
+  return (
+    <Modal closeIcon={<Close />} onClose={onClose} size="tiny">
```

Comment:
I think we're not preventing here the closing of the modal when the `isLoading` property is set, what do you think about adding it?